Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 members = [
-  "compiler/*",
+  "compiler/hash*",
```

Comment:
?

Reply:
Sometimes it is helpful to temporarily remove a crate from the workspace by not prefixing it with `hash-`, otherwise it has to be moved out entirely